src/components/CodeShower.vue

```diff
@@ -232,4 +232,4 @@ return text
     -ms-overflow-style: none;  /* IE and Edge */
     scrollbar-width: none;  /* Firefox */
 } 
-</style>
\ No newline at end of file
+</style>
```

Comment:
Unnecessary new line at EOF.

Reply:
Done. This was added automatically by my os.